site/content/en/docs/Third Party Content/examples.md

```diff
@@ -25,4 +25,16 @@ You will need to download the Minetest client separately to play.
 * [Quilkin](https://github.com/googleforgames/quilkin) is a non-transparent UDP proxy specifically designed for use with large scale multiplayer dedicated game server deployments, to ensure security, access control, telemetry data, metrics and more.
 * [Quilkin with Agones](https://github.com/googleforgames/quilkin/tree/main/examples) is an example of running [Xonotic](https://xonotic.org/) with Quilkin on an Agones cluster, utilising either [the sidecar integration pattern](https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic-sidecar) or via the the [Quilkin xDS Agones provider](https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic-xds) with a TokenRouter to provide routing and access control to the allocated GameServer instance.
 
-You will need to download the Xonotic client to interact with the demo.
\ No newline at end of file
+You will need to download the Xonotic client to interact with the demo.
+
+## Shulker
+
+- [Shulker](https://github.com/jeremylvln/Shulker) is a Kubernetes operator for managing complex and dynamic Minecraft
+  infrastructure, including game servers and proxies.
+- Under the hood, it makes use of Agones to manage the pods lifecycles through `Fleet` and `GameServer` resources.
+- The end user does not manage Agones resources directly but rather simplified abstractions provided by Shulker
+  dedicated for Minecraft workloads.
+- It achieves high availability and density Minecraft infrastructures as long as Agones is just as much.
+
+Shulker requires you to have a genuine Minecraft account. You'll need to purchase the game to test
```

Comment:
Historically, "hacked" Minecraft account were always tolerated. Today, I must say, they represent quite a lot of the player community of Minecraft. It's very common for Minecraft servers to explicitly say that only genuine accounts are accepted. It may be surprising, but I think being explicit here is necessary.

Reply:
Aaah - interesting context! That makes sense, I wasn't aware of that. 

I think a link would be good though, just for context - to here? https://www.minecraft.net/en-us/login ? or here? https://www.minecraft.net/en-us/article/how-create-minecraft-account (for those of us who are not familiar with minecraft).

Reply:
Yup, that's a good idea. Fixed.